stratuslab/python/tar/slipstream_stratuslab/StratusLabIterClientCloud.py

```diff
@@ -103,7 +115,7 @@ def _volume_delete(self, uuid):
         wait_time = 30
         time_stop = time.time() + wait_time
         while 0 != int(pdisk.getValue('count', uuid)):
-            if time.time() <= time_stop:
+            if time.time() >= time_stop:
                 self._print_detail('Disk %s is still in use after waiting for %s sec.' %
                                    (uuid, wait_time))
```

Comment:
Here we will continue waiting even if timeout is expired and the message above will be printed each 3 seconds.
@konstan : Is it the expected behaviour or do we want to exit the loop with a `break` ?